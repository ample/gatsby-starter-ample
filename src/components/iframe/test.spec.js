import React from "react"
import renderer from "react-test-renderer"

import { component as Iframe, fixtures } from "."

describe("Iframe", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Iframe {...fixtures.default} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
