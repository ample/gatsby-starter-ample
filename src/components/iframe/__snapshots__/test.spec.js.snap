// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Iframe renders correctly 1`] = `
<div
  className="iframe_container"
>
  <iframe
    src="/"
    title="FPO iframe"
  />
</div>
`;
